static/css/inbox.less

```diff
@@ -587,17 +589,30 @@ body {
 
     .table-filter {
       float: right;
-      padding-top: 15px;
+      padding-top: 13px;
+      display: flex;
+      font-size: @font-small;
 
       a {
-        padding-left: 6px;
-        padding-right: 6px;
+        padding-left: 0;
+        padding-right: 0;
         color: @grey;
+        text-decoration: none;
 
         &.selected {
           color: #000;
           font-weight: bold;
         }
+
+        &::after {
+          content: ' |'; /* asymmetrical because there's already whitespace after */
+          color: #797979;
```

Comment:
Should this be a variable?

Reply:
It should probably be using the existing variable `@label-color` which is set to `#777777` currently. I'll make that change now...